presto-main/src/main/java/com/facebook/presto/dispatcher/DispatchManager.java

```diff
@@ -127,7 +128,8 @@ public DispatchManager(
             QueryManagerConfig queryManagerConfig,
             DispatchExecutor dispatchExecutor,
             ClusterStatusSender clusterStatusSender,
-            SecurityConfig securityConfig)
+            SecurityConfig securityConfig,
+            Optional<ClusterQueryTrackerService> queryTrackerService)
```

Comment:
please rename this as `clusterQueryTrackerService` for clarity.

Reply:
Can you double check if the code changes are pushed. I still see the old naming of `queryTrackerService`